rollup.config.js

```diff
@@ -2,6 +2,16 @@ import commonjs from '@rollup/plugin-commonjs';
 import { nodeResolve } from '@rollup/plugin-node-resolve';
 import typescript from 'rollup-plugin-typescript2';
 import postcss from 'rollup-plugin-postcss';
+import autoprefixer from "autoprefixer";
```

Comment:
What is it and how it effect?

Reply:
`autoprefixer` postCSS is plugin to parse CSS and add vendor prefixes to CSS rules.